.idea/codeStyles/Project.xml

```diff
@@ -1,6 +1,22 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
     <JetCodeStyleSettings>
+      <option name="PACKAGES_TO_USE_STAR_IMPORTS">
```

Comment:
These are system-level changes. By this I mean these are changes that are specific to android studio and should not be pushed in the PR. Please don't include these changes.